src/cmd_handlers.c

```diff
@@ -1011,6 +1016,44 @@ goto_cmd(const cmd_info_t *cmd_info)
 	return 0;
 }
 
+static int
+regedit_cmd(const cmd_info_t *cmd_info)
+{
+	if (cmd_info->argc > 1)
+		return CMDS_ERR_TRAILING_CHARS;
+
+	const int regname = cmd_info->argc > 0 ? cmd_info->argv[0][0] : DEFAULT_REG_NAME;
+	if (regname == BLACKHOLE_REG_NAME)
+	{
+		ui_sb_err("Cannot modify blackhole register.");
+		return 1;
```

Comment:
Although it's not used consistently at the moment, returning `CMDS_ERR_CUSTOM` in case of error messages is preferred.